x/staking/client/cli/query.go

```diff
@@ -45,7 +44,7 @@ func GetValidatorInfoCmd() *cobra.Command {
 				return err
 			}
 
-			validatorID := hmTypes.ValidatorID(viper.GetInt64(FlagValidatorID))
+			validatorID := viper.GetInt32(FlagValidatorID)
```

Comment:
Do you have any strong reasons to avoid the ValidatorID type?

Reply:
Yes when we query validatorId in routes `validator/{validatorID}` ,  it is throwing invalid type for some reason so i used `int32` in proto